db/seeds.rb

```diff
@@ -0,0 +1,81 @@
+# This file should ensure the existence of records required to run the application in every environment (production,
+# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Example:
+#
+#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
+#     MovieGenre.find_or_create_by!(name: genre_name)
+#   end
+
+# Create Users
+
+user1 = User.create(name: 'John Doe')
+
+user2 = User.create(name: 'Jane Doe')
+
```

Comment:
- [ ] [Optional] [Highly recommended] There is an error raised while running these seeds. Please fix that.

![image](https://github.com/katarighe/recipe-app/assets/68796232/e4dcef32-c846-4ca5-8442-ba790cbc914a)